### PR TITLE
Update README.md file of TFLite Android Example

### PR DESCRIPTION
**Pull request by arp95**
Please check. @perfinion

### CODE REVIEW

File: tensorflow/lite/examples/android/app/README.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ for our external and internal code to merge.
 1. Follow the [Bazel steps for the TF Demo App](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android#bazel):
 
   1. [Install Bazel and Android Prerequisites](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android#install-bazel-and-android-prerequisites).
-     It's easiest with Android Studio.
+     It's easy with Android Studio.
 
-      - You'll need at least SDK version 23.
+      - You'll need atleast SDK version 23.
```
Comment (kyscg) [inline tensorflow/lite/examples/android/app/README.md]: The original version is correct. `At least` is two words indicating the presence of the conjunction 'at' that indicates the bare minimum availability of something.

File: tensorflow/lite/examples/android/app/README.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ for our external and internal code to merge.
 1. Follow the [Bazel steps for the TF Demo App](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android#bazel):
 
   1. [Install Bazel and Android Prerequisites](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android#install-bazel-and-android-prerequisites).
-     It's easiest with Android Studio.
+     It's easy with Android Studio.
```
Comment (kyscg) [inline tensorflow/lite/examples/android/app/README.md]: I think the superlative is better here as Android Studio is best equipped to handle this and hence the 'easiest'

File: tensorflow/lite/examples/android/app/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# TF Lite Android App Example
+# TFLite Android App Example
```
Comment (kyscg) [inline tensorflow/lite/examples/android/app/README.md]: TensorFlowLite ?????

**Review by kyscg — CHANGES_REQUESTED** [review]
Thanks for your contribution, please look at the suggestions

File: tensorflow/lite/examples/android/app/README.md (hunk 2 above)
Comment (arp95) [reply tensorflow/lite/examples/android/app/README.md]: sure changing it.

File: tensorflow/lite/examples/android/app/README.md (hunk 1 above)
Comment (arp95) [reply tensorflow/lite/examples/android/app/README.md]: generally "atleast" is written as a single word. I am changing it to the original.

**Comment by arp95** [comment]
@kyscg please check now.

**Review by kyscg — APPROVED** [review]
Great work @arp95 

All changes look good to me.

**Comment by gbaned** [comment]
@arp95 could you please resolve the conflicts? Thanks!

**Comment by gbaned** [comment]
@arp95 Did you get a chance to check build failures? Please let us know on the update. Thanks!

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**